serialization/avro-serializer/src/main/resources/hazelcast.xml

```diff
@@ -0,0 +1,11 @@
+<hazelcast xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://www.hazelcast.com/schema/config
+                               http://www.hazelcast.com/schema/config/hazelcast-config-3.8.xsd"
```

Comment:
It is already using the latest .xsd version if you don't give the exact name with the version.

https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/config/AbstractXmlConfigHelper.java#L160